pkg/defkinds/php/definition.go

```diff
@@ -112,19 +110,19 @@ type Definition struct {
 // Stage holds all the properties from the base stage that could also be
 // overriden by derived stages.
 type Stage struct {
-	builddef.BaseConfig `mapstructure:",squash"`
-
-	FPM               *bool              `mapstructure:",omitempty"`
-	Command           *string            `mapstrture:"command,omitempty"`
-	Extensions        map[string]string  `mapstructure:"extensions"`
-	ConfigFiles       PHPConfigFiles     `mapstructure:"config_files"`
-	ComposerDumpFlags *ComposerDumpFlags `mapstructure:"composer_dump"`
-	SourceDirs        []string           `mapstructure:"source_dirs"`
-	ExtraScripts      []string           `mapstructure:"extra_scripts"`
-	Integrations      []string           `mapstructure:"integrations"`
-	StatefulDirs      []string           `mapstructure:"stateful_dirs"`
-	Healthcheck       *bool              `mapstructure:"healthcheck"`
-	PostInstall       []string           `mapstructure:"post_install"`
+	ExternalFiles     []llbutils.ExternalFile `mapstructure:"external_files"`
+	SystemPackages    map[string]string       `mapstructure:"system_packages"`
+	FPM               *bool                   `mapstructure:",omitempty"`
+	Command           *string                 `mapstrture:"command,omitempty"`
+	Extensions        map[string]string       `mapstructure:"extensions"`
+	ConfigFiles       PHPConfigFiles          `mapstructure:"config_files"`
+	ComposerDumpFlags *ComposerDumpFlags      `mapstructure:"composer_dump"`
+	SourceDirs        []string                `mapstructure:"source_dirs"`
+	ExtraScripts      []string                `mapstructure:"extra_scripts"`
+	Integrations      []string                `mapstructure:"integrations"`
```

Comment:
about symfony integration, are you sure that symfony always uses the `app` and `src` directory structure ?
IIRC it's not the case anymore since SF4

Reply:
Yep, absolutely! I was trying to fix that when I decided to write the statesolver package :wink: PR #5 actually fix that (sf v3, v4 and v5 are supported) :wink: